packages/apps/transfer/src/services/cross-chain-transfer-finish/finish-xchain-transfer.ts

```diff
@@ -0,0 +1,82 @@
+import { ChainwebNetworkId } from '@kadena/chainweb-node-client';
+import { ContCommand, getContCommand } from '@kadena/client';
+import { ChainId } from '@kadena/types';
+
+import { generateApiHost } from '../utils/utils';
+
+interface ITransactionData {
+  sender: { chain: ChainId; account: string };
+  receiver: { chain: ChainId; account: string };
+  amount: number;
+  receiverGuard: {
+    pred: string;
+    keys: [string];
+  };
+}
+export interface ITransferDataResult {
+  tx?: ITransactionData | undefined;
+  error?: string;
+}
+
+export interface TransferResult {
+  requestKey?: string;
+  status?: string;
+}
+
+const gasLimit: number = 850;
+const gasPrice: number = 0.00000001;
```

Comment:
The gas limit already is an env var, but the price it think it may change regarding the speed. We may need to give the user an option to increase it if they want better speed (like the option in Chainweaver).